srcpkgs/exa/template

```diff
@@ -1,21 +1,24 @@
 # Template file for 'exa'
 pkgname=exa
-version=0.9.0
-revision=2
+version=0.10.0
+revision=1
 build_style=cargo
-hostmakedepends="pkg-config"
+hostmakedepends="pkg-config libgit2-devel"
 makedepends="libgit2-devel"
 short_desc="Modern replacement for ls"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="FollieHiyuki <folliekazetani@protonmail.com>"
 license="MIT"
 homepage="https://the.exa.website/"
-distfiles="https://github.com/ogham/exa/archive/v${version}.tar.gz"
-checksum=@facbe3b234f403e1e1919f28f009f1733a0fbbbb11f0bc2ba89ac3eba11bc5e8
+distfiles="https://github.com/ogham/exa/archive/v${version}.tar.gz
+ https://github.com/ogham/exa/releases/download/v${version}/exa-accoutrements-v${version}.zip"
+checksum="27420f7b805941988399d63f388be4f6077eee94a505bf01c2fb0e7d15cbf78d
+ c1ab340af63e64bc3fd854c03f6161aa240a533e928688036a7d4544aecabc05"
 
 post_install() {
-	vman contrib/man/exa.1
-	vcompletion contrib/completions.bash bash
-	vcompletion contrib/completions.fish fish
-	vcompletion contrib/completions.zsh zsh
+	vman ../man/exa.1
+	vman ../man/exa_colors.5
```

Comment:
I much prefer this over generating them at build time (especially when it uses pandoc).